Update/omot_026.txt

```diff
@@ -6700,7 +6700,7 @@ void main()
 	//r自分でも信じていないことを教え諭すのは、無意味どころか偽善的な行為だ。だから私たち二人は、無限に続く時の繰り返しをただ漫然と過ごすようになってしまったのだ…。
 		if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 		OutputLine(NULL, "自分でも信じていないことを教え諭すのは、無意味どころか偽善的な行為だ。だから私たち二人は、無限に続く時の繰り返しをただ漫然と過ごすようになってしまったのだ…。",
-				NULL, "Teaching her something I didn't believe in myself wouldn't just be meaningless, it would be hypocritical. That's why the two of us just endlessly continued this repeating cycle...", Line_Normal);
+				NULL, "Teaching her something I didn't believe in myself wouldn't just be meaningless, it would be hypocritical. That's why the two of us just desultorily continued this repeating cycle...", Line_Normal);
```

Comment:
```suggestion
				NULL, "Teaching her something I didn't believe in myself wouldn't just be meaningless, it would be hypocritical. That's why the two of us just aimlessly continued this repeating cycle...", Line_Normal);
```